### PR TITLE
Add Postal Address checkbox to Partner Location

### PR DESCRIPTION
**Pull request by riadmen**
#16844

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5712360_sys_gh16844_Add_IsPostalAddress_Partner_Location.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
+-- 2023-12-01T13:54:50.587Z
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,582828,0,'IsPostalAddress',TO_TIMESTAMP('2023-12-01 14:54:49','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','Postadresse','Postadresse',TO_TIMESTAMP('2023-12-01 14:54:49','YYYY-MM-DD HH24:MI:SS'),100)
```
Comment (cristinamghita) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5712360_sys_gh16844_Add_IsPostalAddress_Partner_Location.sql:2]: This does not belong in metasfresh repo.
Is a column added for this customer.

**Review by cristinamghita — CHANGES_REQUESTED** [review]
check my comments

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5712360_sys_gh16844_Add_IsPostalAddress_Partner_Location.sql (hunk 1 above)
Comment (riadmen) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5712360_sys_gh16844_Add_IsPostalAddress_Partner_Location.sql:2]: - There is already a customer windows (540905 - standard is 123)
- IMHO `IsShipTo` or `IsHandoverLocation` has a meaning very far away from `IsPostalAddress` is 
- Ok i will move it to customer repo

Comment (cristinamghita) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5712360_sys_gh16844_Add_IsPostalAddress_Partner_Location.sql:2]: The org partner are maintained from the Dist Org window and not from regular partner window.
This is the correct window ID: 540676

**Comment by teosarca** [comment]
closing, legacy

**Pull request closed without merging**